package.json

```diff
@@ -29,7 +29,7 @@
     "test": "grunt test && cd external && cd amber-cli && npm test && cd .. && cd .."
   },
   "devDependencies": {
-    "amber-dev": "^0.3.0",
+    "amber-dev": "^0.3.1",
```

Comment:
Not needed, really. Worked fine with 0.3.0 and anyone getting the clone or doing `bower install` gets 0.3.1 (or newer 0.3.x) anyway. Don't include.